core/src/main/scala/org/genivi/sota/core/PackagesResource.scala

```diff
@@ -88,17 +89,18 @@ class PackagesResource(resolver: ExternalResolverClient,
     * An ota client GET [[Package]] info, including for example S3 uri of its binary file.
     */
   def fetch(ns: Namespace, pid: PackageId): Route = {
-    // TODO: Include error description with rejectEmptyResponse?
-    rejectEmptyResponse {
-      complete {
-        val query = for {
-          p <- Packages.byId(ns, pid)
-          isBlacklisted <- BlacklistedPackages.isBlacklisted(ns, pid)
-        } yield (p, isBlacklisted)
-
-        db.run(query).map { case (pkg, blacklisted) =>
-          pkg.toResponse(blacklisted)
-        }
+    complete {
```

Comment:
Is this the best place we have to measure bandwidth? I would think that `downloadPackage` in DeviceUpdatesResources.scala would be better, since that is where a device goes to download packages.